ehitchhiking-ui/src/app/shared/components/rate-passangers-modal/rate-passangers-modal.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component} from '@angular/core';
+import {MatDialogRef} from '@angular/material';
+import {Passanger} from './passanger';
+
+@Component({
+  selector: 'rate-passangers-modal',
+  templateUrl: './rate-passangers-modal.component.html',
+  styleUrls: ['./rate-passangers-modal.component.sass']
+})
+export class RatePassangersModalComponent {
+
+  passangers: Passanger[] = [
+    {id: 0, name: 'Ivan', rating: 0},
+    {id: 1, name: 'Egor', rating: 0},
+    {id: 2, name: 'Semen', rating: 0},
+    {id: 3, name: 'Fedor', rating: 0},
+  ];
+
+  constructor( public dialogRef: MatDialogRef<RatePassangersModalComponent>) {}
+
+  rateUser(clickObj): void {
```

Comment:
type?

Reply:
There should be emitValue. Pls remember that you should change that name

Reply:
fixed